modules/oe_translation_epoetry/src/Plugin/Field/FieldType/ContactItem.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation_epoetry\Plugin\Field\FieldType;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\TypedData\DataDefinition;
+
+/**
+ * Defines the 'oe_transalation_epoetry_contact' field type.
+ *
+ * @FieldType(
+ *   id = "oe_transalation_epoetry_contact",
+ *   label = @Translation("Contact"),
+ *   category = @Translation("OpenEuropa"),
+ *   default_widget = "oe_transalation_epoetry_contact_widget",
+ *   default_formatter = "oe_transalation_epoetry_contact_formatter"
+ * )
+ */
+class ContactItem extends FieldItemBase implements ContactItemInterface {
+
+  /**
+   * Value for the 'contact_type' setting: Requester.
+   */
+  const REQUESTER = 'Requester';
```

Comment:
If you made an interface, you can go ahead and put these constants on the interface.